tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -62,37 +111,45 @@ def test_no_matches(self):
         code_mappings = self.code_mapping_helper.generate_code_mappings(stacktraces)
         assert code_mappings == []
 
-    def test_more_than_one_match_does_not_derive(self):
+    def test_more_than_one_match_does_derive(self):
         stacktraces = [
-            # More than one file matches for this, thus, no stack traces will be produced
+            # More than one file matches for this, however, the package name is taken into account
             # - "src/sentry_plugins/slack/client.py",
             # - "src/sentry/integrations/slack/client.py",
             "sentry_plugins/slack/client.py",
         ]
         code_mappings = self.code_mapping_helper.generate_code_mappings(stacktraces)
+        assert code_mappings == [
+            CodeMapping(
+                repo=self.foo_repo,
+                stacktrace_root="sentry_plugins",
+                source_path="src/sentry_plugins",
+            )
+        ]
+
+    def test_no_stacktraces_to_process(self):
+        code_mappings = self.code_mapping_helper.generate_code_mappings([])
         assert code_mappings == []
 
     def test_more_than_one_match_works_when_code_mapping_excludes_other_match(self):
         stacktraces = [
             "sentry/identity/oauth2.py",
-            # This file matches two files in the repo, however, because we first
-            # derive the sentry code mapping we can exclude one of the files
             "sentry_plugins/slack/client.py",
         ]
         code_mappings = self.code_mapping_helper.generate_code_mappings(stacktraces)
         assert code_mappings == self.expected_code_mappings
 
     def test_more_than_one_match_works_with_different_order(self):
-        # We do *not* derive sentry_plugins because we don't derive sentry first
         stacktraces = [
             # This file matches twice files in the repo, however, the reprocessing
             # feature allows deriving both code mappings
             "sentry_plugins/slack/client.py",
             "sentry/identity/oauth2.py",
         ]
         code_mappings = self.code_mapping_helper.generate_code_mappings(stacktraces)
-        # Order matters, this is why we only derive one of the two code mappings
-        assert code_mappings == self.expected_code_mappings
+        # Asserting lists with different order of elements would fail, thus, checking indexes
+        assert code_mappings[0] == self.expected_code_mappings[1]
+        assert code_mappings[1] == self.expected_code_mappings[0]
```

Comment:
Before it worked because the Reprocessing of stacktraces would allow matching `sentry_plugins` after the `sentry` code mappings would be generated.

We could make the call of `generate_code_mappings` to always return code mappings in the same order. I think this is fine.